airflow/www/static/js/utils/index.ts

```diff
@@ -185,6 +185,34 @@ const toSentenceCase = (camelCase: string): string => {
   return "";
 };
 
+const addColorKeyword = (
```

Comment:
If the function is separated-out into a utility, can you use this opportunity to add a unit test to airflow/www/static/js/utils/index.test.ts as well?

Reply:
Alongside, the name of the function might be mis-leading for first-time readers. Could you rename it to `coloByKeywords()` or `highlightByKeywords()`?

Reply:
Thanks added tests and renamed function to `highlightByKeywords` .